metatron-app/metatron-discovery
# #3726 Adjust horizontal scroll in grid and Fixed an issue where zeros were not displayed

### Description
Adjust horizontal scroll in grid and Fixed an issue where zeros were not displayed

**Related Issue** : #3726 


### How Has This Been Tested?

1. Create a grid.
2. Check that horizontal scrolling is displayed normally.
3. Verify that 0 is displayed.

#### Need additional checks?

### Types of changes



### Checklist:


### Additional Context
